e2e/updatecli.d/success.d/gitBranch.yaml

```diff
@@ -39,7 +39,7 @@ targets:
   #  disablesourceinput: true
   #  spec:
   #    branch: v1
-  alreadyexist:
+  alreadyexists:
```

Comment:
I'm assuming this is merely a label. This is more of a grammatical change than a spelling change.